canonicalwebteam/blog/wordpress.py

```diff
@@ -1,3 +1,4 @@
+import flask
```

Comment:
wordpress is a model, so framework agnostic.
This should be handled somewhere else.

Reply:
Do you mean handling it in the `notFoundError` class ass an exception?

Reply:
I mean that this class doesn't need to have a `flask` dependency. It's some kind of model, so should be as light as possible. This class just manages the requests to the wordpress API.